provider/oauth2/views.py

```diff
@@ -8,13 +8,11 @@
 
 from provider import constants
 from provider.oauth2.backends import BasicClientBackend, RequestParamsClientBackend, PublicPasswordBackend
-from provider.oauth2.forms import AuthorizationCodeGrantForm
-from provider.oauth2.forms import AuthorizationRequestForm, AuthorizationForm
-from provider.oauth2.forms import PasswordGrantForm, RefreshTokenGrantForm
+from provider.oauth2.forms import (AuthorizationCodeGrantForm, AuthorizationRequestForm, AuthorizationForm,
+                                   PasswordGrantForm, RefreshTokenGrantForm, ClientCredentialsGrantForm)
 from provider.oauth2.models import Client, RefreshToken, AccessToken
 from provider.utils import now
-from provider.views import AccessToken as AccessTokenView, OAuthError, AccessTokenMixin
-from provider.views import Capture, Authorize, Redirect
+from provider.views import AccessToken as AccessTokenView, OAuthError, AccessTokenMixin, Capture, Authorize, Redirect
```

Comment:
I'll take "not touching the code we want to kill" for 400, please, Alex.

I seriously don't even want to make this PR. I would much rather invest the time/energy in getting us to django-oauth-toolkit. This works. It's tested. I'm not touching it. If anyone feels strongly about renaming, submit another PR.